Pear/Profile/EditProfileSectionsViewController.swift

```diff
@@ -96,44 +130,95 @@ class EditProfileSectionsViewController: UIViewController {
             guard let cell = fadeTableView.view.dequeueReusableCell(withIdentifier: EditProfileSectionTableViewCell.reuseIdentifier) as? EditProfileSectionTableViewCell, indexPath.row < interests.count else { return addProfileSectionTableViewCell(tableView, cellForRowAt: indexPath)
             }
-            cell.configure(with: interests[indexPath.row])
+            cell.configure(with: interests[indexPath.row], user: updatingUser, index: indexPath.row, delegate: self)
             return cell
-        case .groups(let groups):
-            guard let cell = fadeTableView.view.dequeueReusableCell(withIdentifier: EditProfileSectionTableViewCell.reuseIdentifier) as? EditProfileSectionTableViewCell, indexPath.row < groups.count else { return addProfileSectionTableViewCell(tableView, cellForRowAt: indexPath)
+            
+        case .groups(_):
+            guard let cell = fadeTableView.view.dequeueReusableCell(withIdentifier: EditProfileSectionTableViewCell.reuseIdentifier) as? EditProfileSectionTableViewCell,
+                    indexPath.row < groups.count else { return addProfileSectionTableViewCell(tableView, cellForRowAt: indexPath)
             }
-            cell.configure(with: groups[indexPath.row])
+            cell.configure(with: groups[indexPath.row], user: updatingUser, index: indexPath.row, delegate: self)
             return cell
-        case .prompts(let prompts):
+        case .prompts(_):
             guard let cell = tableView.dequeueReusableCell(withIdentifier: PromptTableViewCell.reuseIdentifier, for: indexPath) as? PromptTableViewCell else { return UITableViewCell() }
             cell.configure(for: prompts[indexPath.row])
-            cell.removePrompt = { [weak self] selectedCell in
-                print("remove")
+                cell.removePrompt = { [weak self] selectedCell in
+                    guard let self = self else { return }
+                    guard let indexPath = self.fadeTableView.view.indexPath(for: cell) else { return }
+                    self.prompts[indexPath.row] = Prompt(id: nil, questionName: "", questionPlaceholder: "", answer: nil)
+                    /// Pass this indexOfPrompt into the updating VC (when it's pushed) and then when passed back from the delegate, we need to update the self.prompts array at this specific index
+                    self.indexOfPrompt = indexPath.row
+                }
+            NetworkManager.getPromptOptions { prompts in
+                DispatchQueue.main.async {
+                    self.promptOptions = prompts.filter { !self.prompts.contains($0) }
+                    self.fadeTableView.view.reloadData()
+                }
             }
             return cell
         }
     }
     
+    /// If user selects the last row of the section, we need to push the adding controller
+    func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
+        switch editProfileSectionType {
+        case .interests(_):
```

Comment:
`case .interests:` Basically just remove the `(_)` when you don't need to access to the associate value

Reply:
Do this for the rest 🙏